cmd/transflect-operator/main.go

```diff
@@ -22,11 +22,12 @@ var (
 )
 
 type config struct {
-	UseIngress bool             `short:"i" help:"Create and use temporary ingress to access temporary service, e.g. from outside cluster"`
-	Address    string           `short:"a" help:"gRPC reflection server address, host:port"`
-	Plaintext  bool             `short:"p" help:"Use plain-text; no TLS"`
-	LogFormat  string           `help:"Log format ('json', 'std')" enum:"json,std" default:"std"`
-	Version    kong.VersionFlag `short:"v" help:"Print version information"`
+	UseIngress     bool             `short:"i" help:"Create and use temporary ingress to access temporary service, e.g. from outside cluster"`
+	Address        string           `short:"a" help:"gRPC reflection server address, host:port"`
+	Plaintext      bool             `short:"p" help:"Use plain-text; no TLS"`
+	LeaseNamespace string           `help:"Namespace in which leader election lease is created" default:"transflect"`
```

Comment:
If you give this a env var too (LEASE_NAMESPACE), you can add the following to your deployment.yaml to make this automatically wherever the pod is running:
```
containers:
- name: transflect
  env:
  - name: LEASE_NAMESPACE
    valueFrom: { fieldRef: { fieldPath: metadata.namespace } }
```

I'm not sure how you could use a fieldRef in the command args, so an env var is probably needed for this technique.

Reply:
that's neat!